sw/device/lib/dif/dif_usbdev.c

```diff
@@ -220,6 +220,16 @@ dif_result_t dif_usbdev_configure(const dif_usbdev_t *usbdev,
     return kDifError;
   }
 
+  // TODO: in dvsim the packet memory contents are uninitialized and there is
+  // an 'X' checker on the TL-UL bus meaning that we cannot read/write a partial
+  // word
```

Comment:
I don't have a problem with a software approach necessarily, but is there a way to initialize the RAM via the DV environment?

If we're doing it the software way, we'll need to adjust the DIF unit tests to expect these memory transactions as well.